product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/01_vpc_endpoint.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: VPC endpoint example
+
+---
+
+This example shows how to connect your cluster using VPC endpoints.
+
+Assume that your cluster is on a account called `development` and is being accessed from a Linux client VM on another account called `test` with the following properties:
+
+-   Cluster:
+    -   AWS account: `development`
+    -   Amazon resource name (ARN): `arn:aws:iam::123456789123:root`
+    -   Cluster ID: `p-c4j0jfcmp3af2ieok5eg` 
+    -   Account ID: `brcxzr08qr7rbei1` 
+    -   Organization's domain name: `biganimal.io` 
+
+
+-   Linux client VM called `vm-client`:
+    -   AWS account: `test`
+    -   Resource group: `rg-client`
+    -   VPC: `vpc-client`
+    -   VPC subnet: `snet-client`
+
+## Prerequisites
+
+To walk through an example in your own environment, you need:
+
+- Your cluster URL. You can find the URL in the Connect tab of your cluster instance in the BigAnimal portal.
+ 
+
+## Step 1: Create an endpoint service for your cluster
```

Comment:
This section should probably clearly indicate which account the actions are to be performed in.